sdk/agrifood/agrifood-farming-rest/sample.env

```diff
@@ -2,11 +2,11 @@
 #
 # See the documentation for `EnvironmentCredential` at the following link:
 # https://docs.microsoft.com/javascript/api/@azure/identity/environmentcredential
-AZURE_TENANT_ID=<AD tenant id or name>
-AZURE_CLIENT_ID=<ID of the user/service principal to authenticate as>
-AZURE_CLIENT_SECRET=<client secret used to authenticate to Azure AD>
-
+AZURE_TENANT_ID=72f988bf-86f1-41af-91ab-2d7cd011db47
+AZURE_CLIENT_ID=39d176c5-91b6-454b-854d-55813c17d92d
+AZURE_CLIENT_SECRET=TOo8Q~_bZFdCLspdxJLrh-zRCCpYrf56-2D4FbJ5
+TEST_MODE=record
```

Comment:
This file shouldn't be changed. You need to create a .env file locally that gitignore will ignore.